packages/perseus/src/widgets/input-number/input-number.test.ts

```diff
@@ -12,12 +12,13 @@ import {mockStrings} from "../../strings";
 import {renderQuestion} from "../__testutils__/renderQuestion";
 
 import InputNumber from "./input-number";
+import inputNumberValidator from "./input-number-validator";
 import {question3 as question} from "./input-number.testdata";
 
 import type {
     PerseusInputNumberWidgetOptions,
     PerseusRenderer,
-} from "../../perseus-types";
+} from "@khanacademy/perseus";
```

Comment:
That's weird, I think this should have been a linting error. 🤔 I think we have a linter that prevents packages from importing from themselves like this.

I don't know why the linter isn't mad, but I would revert this.

Reply:
Already reverted it ^_^ Thank you! Trying to keep an eye out for things like this after you pointed out that strings import issue